atomic_reactor/plugins/pre_pin_operator_digest.py

```diff
@@ -151,9 +163,20 @@ def _get_operator_manifest(self):
         return operator_manifest
 
     def _get_pullspecs(self, operator_manifest):
+        self.log.info("Looking for pullspecs in operator CSV files")
         pullspec_set = set()
+
         for operator_csv in operator_manifest.files:
-            pullspec_set.update(operator_csv.get_pullspecs())
+            if not operator_csv.has_related_images():
+                self.log.info("Getting pullspecs from %s", operator_csv.path)
+                pullspec_set.update(operator_csv.get_pullspecs())
+            elif operator_csv.has_related_image_envs():
+                msg = ("Both relatedImages and RELATED_IMAGE_* env vars present in {}. "
```

Comment:
Nope, we want opposite here. Keeping RELATED_IMAGE_ and letting OSBS to construct relatedImages section

Reply:
@lcarva commented in ticket:
> if relatedImages is set and there are RELATED_IMAGE_* environment variables, fail the build. This is to avoid confusion and ambiguous behavior.

Is that not what we want?

Reply:
Oh you meant the error message right?

Reply:
yes, message should be different

Reply:
Ok, fixed in last force-push